### ISSUE TITLE
Teleportation to starting point during /jump

### ISSUE THREAD

**Issue by SophiaLVP** [LVPlayground/playground#621]
Today I was roaming /jump with Halsey and I observed something. When you fail a jump and want to try again, you /leave the jump, find your car again (cars other than infurnus, elegy, turismo, sul and many more) and then rejoin the jump using command. I think it's a bit odd for me. There should be a command like /jr, /jreset or /jumpreset which will spawn you back to the starting point of jump without leaving it. If jump is missed, use the command to try again. It will help a lot.

What do you think?

**Comment by Specifer** [comment]
It's just as same as leave and start again?It can be done... Yet still does it actually need to be done ?

**Comment by SophiaLVP** [comment]
Yes, that command is a big deal for me and can save us from stress of leaving, spawning a car, joining, leaving, spawning a car, joining, leaving, spawning a car, joining and so on.

**Comment by Specifer** [comment]
I guess it can't be done :/ .
